cnd/src/http_api/action.rs

```diff
@@ -34,7 +34,8 @@ pub enum ActionExecutionParameters {
 pub enum ActionResponseBody {
     BitcoinSendAmountToAddress {
         to: bitcoin_support::Address,
-        amount: bitcoin_support::BitcoinQuantity,
+        #[serde(with = "bitcoin_support::amount::serde::as_sat")]
```

Comment:
Need to add a test here as I assume we use to serialize as a string and now we do an int? Need to double check.